assets/data-port/import/done/done-page.js

```diff
@@ -1,88 +1,97 @@
+					</Section>
+				</section>
+			) }
 
-	const hasErrors = resultErrors.some( ( [ , count ] ) => count );
-	const hasSuccess = resultSuccess.some( ( [ , count ] ) => count );
+			{ logs.notice && logs.notice.length > 0 && (
+				<section className="sensei-data-port-step">
+					<Section className="sensei-data-port-step__body">
+						<h2>{ __( 'Partial', 'sensei-lms' ) }</h2>
+						<p className="sensei-import-done__section-description">
+							{ __(
+								'The following content was partially imported. The import process encountered some issues that you can resolve manually by clicking the link and making the necessary adjustments.',
+								'sensei-lms'
+							) }
+						</p>
 
-	return (
-		<section className="sensei-data-port-step sensei-import-done">
-			<Section className="sensei-data-port-step__body">
-				<section className="sensei-import-done__result">
-					{ hasErrors ? (
-						<Dashicon
-							icon="warning"
-							size={ 150 }
-							className="sensei-import-done__result-icon--warning"
-						/>
-					) : (
-						<Dashicon
-							icon="yes-alt"
-							size={ 150 }
-							className="sensei-import-done__result-icon--success"
-						/>
-					) }
-					{ hasSuccess && (
-						<>
-							<p>
-								{ __(
-									'The following content was imported:',
-									'sensei-lms'
-								) }
-							</p>
-							<ImportResults entries={ resultSuccess } />
-						</>
-					) }
+						<ImportLog items={ logs.notice } type="warning" />
+					</Section>
 				</section>
-				{ hasErrors && (
-					<>
-						<section className="sensei-import-done__errors">
-							<p>
-								{ formatString(
-									__(
-										'The following content {{strong}}failed{{/strong}} to import:',
-										'sensei-lms'
-									)
-								) }
-							</p>
-							<ImportResults entries={ resultErrors } />
-						</section>
-						<section className="sensei-import-done__log">
-							<ImportLog
-								result={ logs }
-								reload={ fetchImportLog }
-							/>
-						</section>
-					</>
-				) }
+			) }
+		</>
+	);
+
+	if ( isFetching ) {
+		logsElement = (
+			<div className="sensei-import-done__log-fetching">
+				<Spinner /> { __( 'Fetching log details…', 'sensei-lms' ) }
+			</div>
+		);
+	} else if ( fetchError ) {
+		logsElement = (
+			<Notice status="error" isDismissible={ false }>
+				{ __( 'Failed to load import log.', 'sensei-lms' ) }{ ' ' }
+				{ fetchError.message }
+				<Button onClick={ retry } isLink isSmall>
+					{ __( 'Retry', 'sensei-lms' ) }
+				</Button>
+			</Notice>
+		);
+	}
+
+	return (
+		<>
+			<section className="sensei-data-port-step">
+				<Section className="sensei-data-port-step__body">
+					<h2>{ __( 'Completed', 'sensei-lms' ) }</h2>
+					<p className="sensei-import-done__section-description">
+						{ __(
+							'The following content was imported:',
+							'sensei-lms'
+						) }
+					</p>
+
+					<ImportSuccess successResults={ successResults } />
```

Comment:
I'd still keep this named something like Result/Summary, we only use it for the success block now, but the component itself just renders a summary list.

Reply:
I renamed it to `ImportSuccessResults`: https://github.com/Automattic/sensei/pull/3379/commits/dcca5ad540689862ff2524bff074a182768b8544
WDYT?